core/modules/ccontrol/UserQuerySelect.h

```diff
@@ -178,7 +178,7 @@ class UserQuerySelect : public UserQuery {
     std::string _queryIdStr{QueryIdHelper::makeIdStr(0, true)};
     bool _killed{false};
     std::mutex _killMutex;
-    std::string _errorExtra;    ///< Additional error information
+    mutable std::string _errorExtra;    ///< Additional error information
```

Comment:
so the error can be set from a const method

Reply:
We should really switch to more reasonable error reporting that does not need this internal state variable.